From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 08 Jun 2016 21:04:00 +0200
Subject: [PATCH] Do not use -rpath

Hard-coded rpath is bad, moreover when it contains absolute path to the
build directory.
--- a/SuiteSparse_config/SuiteSparse_config.mk
+++ b/SuiteSparse_config/SuiteSparse_config.mk
@@ -338,7 +338,7 @@
 
     ifeq ($(UNAME),Linux)
         # add the realtime library, librt, and SuiteSparse/lib
-        LDLIBS += -lrt -Wl,-rpath=$(INSTALL_LIB)
+        LDLIBS += -lrt
     endif
 
     #---------------------------------------------------------------------------
